interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -223,6 +222,27 @@ namespace cling {
     std::unique_ptr<InterpreterCallbacks>
        AutoLoadCB(new AutoloadCallback(this, showSuggestions));
     setCallbacks(std::move(AutoLoadCB));
+
+  }
+
+  Interpreter::Interpreter(int argc, const char* const *argv,
```

Comment:
For constructor delegation, these two delegating constructors should be inlined.